test/eager/test_eager_with_xla_compile.py

```diff
@@ -45,21 +51,34 @@ def test_eager_with_compile_basic(self):
     # no egaer execution should happen inside this compiled graph
     self.assertNotIn("EagerOpExecuteTime", met.metric_names())
 
+  def test_eager_execute_compiled_multiple_times(self):
+    met.clear_all()
+    self.assertTrue(torch_xla.experimental.is_eager_mode())
+    device = torch_xla.device()
+    # this part happens eagerly
+    t1 = torch.randn(10, 5, device=device)
+    t1.add_(0.5)
+    compiled = torch_xla.compile(self.dummy_cos_sin)
+    res = compiled(compiled(t1))
+    self.assertTrue(
+        torch.allclose(res * 0.3,
+                       self.dummy_cos_sin(self.dummy_cos_sin(t1)) * 0.3))
+    xm.wait_device_ops()
+    self.assertEqual(met.metric_data("ExecuteTime")[0], 2)
+
+  def test_eager_with_compile_graph_break(self):
+    met.clear_all()
+    self.assertTrue(torch_xla.experimental.is_eager_mode())
+    device = torch_xla.device()
+    t1 = torch.randn(5, 5, device=device)
 
-def test_eager_execute_compiled_multiple_times(self):
-  met.clear_all()
-  self.assertTrue(torch_xla.experimental.is_eager_mode())
-  device = torch_xla.device()
-  # this part happens eagerly
-  t1 = torch.randn(10, 5, device=device)
-  t1.add_(0.5)
-  compiled = torch_xla.compile(self.dummy_cos_sin)
-  res = compiled(compiled(t1))
-  self.assertTrue(
-      torch.allclose(res * 0.3,
-                     self.dummy_cos_sin(self.dummy_cos_sin(t1)) * 0.3))
-  xm.wait_device_ops()
-  self.assertEqual(met.metric_data("ExecuteTime")[0], 2)
+    with self.assertRaises(Exception) as context:
+      t2_compiled = torch_xla.compile(
+          self.dummy_graph_break, full_graph=True)(
+              t1)
+    self.assertIn(
```

Comment:
`assertRaisesRegex` is what you want here https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaisesRegex